### PR TITLE
Added "More Lenses" mod

### PR DESCRIPTION
**Pull request by astog**
Combined code from original CQUI and my "More Lenses" mod.

### CODE REVIEW

**Comment by chaorace** [comment]
Checking this out now, thanks for contributing!

File: qui.modinfo [hunk 1]
```diff
@@ -94,6 +94,24 @@
                 <File>URS/UnitPromotionPopup.lua</File>
             </Items>
         </ImportFiles>
+		<ImportFiles id="MORELENSES_IMPORT_FILES">
```
Comment (chaorace) [inline qui.modinfo]: The default indent character for CQUI is a double space, please don't use tabs when adding to CQUI modfiles

**Comment by chaorace** [comment]
@astog Changes look good except for that style issue, I can fix it on my end with a quick cherry-pick/amend, but I'd prefer being able to merge your PR so the commit history makes more sense. If you can please fix the indent issue in qui.modinfo and I'll merge right away!

File: qui.modinfo (hunk 1 above)
Comment (astog) [inline qui.modinfo]: Sorry, missed that. Will fix that right away.

**Comment by astog** [comment]
@chaorace Fixed the indentation. Let me know if everything else is fine.

**Comment by chaorace** [comment]
Agh! I'm sorry it's taken me way too long to merge your PRs! Thanks so much for contributing and I apologize for not paying your work my full attention. Merging now that the style issue is resolved

**Pull request merged**